packages/driver/src/cypress/cy.ts

```diff
@@ -1096,27 +1084,6 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
     return this.Cypress.action('app:navigation:changed', `page navigation event (${event})`)
   }
 
-  private cleanup () {
```

Comment:
The command_queue handles this clean up now since it's setting & managing these values.